services/worker-manager/test/provider_aws_test.js

```diff
@@ -497,6 +497,24 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
       assert.strictEqual(provider.seen[worker.workerPoolId], 0);
     });
 
+    test('update expiration for a long-running worker', async function() {
+      const expires = taskcluster.fromNow('-1 week');
+      fake.rgn('us-west-2').instanceStatuses['i-123'] = 'terminated';
```

Comment:
What is the reason to define the instance as terminated here?

Reply:
I had the same question :)

Reply:
Ah, this is from the intermediate commit, I think this was a copy paste or commit fail, it should have been made running (and was at some point!?) - the instance status had to be defined.